store/models/log_events.go

```diff
@@ -274,38 +269,89 @@ func (le RunLogEvent) Requester() common.Address {
 	return common.BytesToAddress(b)
 }
 
+func (le RunLogEvent) Payment() common.Address {
+	b := le.Log.Topics[RequestLogTopicRequester].Bytes()
+	return common.BytesToAddress(b)
+}
+
 // JSON decodes the CBOR in the ABI of the log event.
 func (le RunLogEvent) JSON() (JSON, error) {
-	el := le.Log
-	js, err := decodeABIToJSON(el.Data)
+	//el := le.Log
+	//js, err := decodeABIToJSON(el.Data)
+	//if err != nil {
+	//return js, err
+	//}
+
+	topic, err := le.getTopic(0)
+	if err != nil {
+		return JSON{}, err
+	}
+	parse, ok := topicFactoryMap[topic]
+	if !ok {
+		return JSON{}, fmt.Errorf("No parser for the RunLogEvent topic %v", topic)
+	}
+	return parse(le)
+}
+
+var evmWordSize = common.HashLength
+var idSize = evmWordSize
```

Comment:
`common.HashLength` is a `const`. Therefore these can also be a `const`:

```
// Lengths of hashes and addresses in bytes.
const (
	// HashLength is the expected length of the hash
	HashLength = 32
	// AddressLength is the expected length of the address
	AddressLength = 20
)
```
https://github.com/ethereum/go-ethereum/blob/master/common/types.go#L34-L38